docs/source/developers/guide/documentation.rst

```diff
@@ -0,0 +1,30 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+
+.. SCOPE OF THIS SECTION
+.. To expose that help with existing documentation is a
+.. very good way to start and also a very important part of
+.. the project! If possible add summary of the structure of
+.. the existing documentation, including different Cookbooks.
+
+
+.. _documentation:
+
+**************************
+Helping with documentation
+**************************
```

Comment:
What is the plan for filling this page? Depending on what we add to this page we may also want to move it to the upper "developers" section.

Reply:
From the description:

```
.. SCOPE OF THIS SECTION
.. To expose that help with existing documentation is a
.. very good way to start and also a very important part of
.. the project! If possible add summary of the structure of
.. the existing documentation, including different Cookbooks.
```

Reply:
Agree, I am still not sure where to put it. Can we leave it here and get back to it, when I will create a PR with the content?

Reply:
> Can we leave it here and get back to it, when I will create a PR with the content?

Yes, that's what I would suggest